dnd-ui/webview-ui/src/nodes/ButtonNode.tsx

```diff
@@ -54,9 +55,9 @@ const ButtonNode = (props) => {
       <Handle
         type="target"
         position={Position.Left}
-        style={{ background: "#555", width: 10, height: 10  }}
-        id={sourceHandleId || uuid()}
-        onConnect={(params) => console.log("handle onConnect", params)}
+        style={{ background: '#555', width: 10, height: 10 }}
+        id={sourceHandleId}
```

Comment:
When a new node is created, without `uuid()` call here the `Handle` would be null